packages/smooth_app/lib/data_models/github_contributors_model.dart

```diff
@@ -0,0 +1,10 @@
+class ContributorsModel {
+  ContributorsModel({required this.avatarUrl, required this.profilePath});
+
+  ContributorsModel.fromJson(Map<String, dynamic> json)
```

Comment:
```suggestion
  ContributorsModel.fromJson(Map<String, String> json)
```
You cast to String anyways

Reply:
ContributorsModel.fromJson(Map<String, String> json)
      : profilePath = json['html_url'] ?? '',
        avatarUrl = json['avatar_url'] ?? '';

Reply:
Map<String, dynamic> should be fine here